server/products.js

```diff
@@ -17,17 +22,71 @@ module.exports = require('express').Router()
   //  only admins should be able to create products
   .post('/', (req, res, next) => {
     Product.create(req.body)
-    .then(product => res.json(product)
-    .catch(next))
+    .then(product => res.status(201).json(product))
+    .catch(next)
   })
```

Comment:
consider taking out some of these comments, they really clog up the space a little bit (unless theyre actually useful)